### ISSUE TITLE
Footer Container on NAAS search Page

### ISSUE THREAD

**Issue by richagarg278** [jupyter-naas/awesome-notebooks#1906]
**Describe the bug**
Footer on this Page needs a background of #2C2C2C as designed in the Image below and try to maintain left and right padding also.
![SEARCH - PUBLIC NOT LOGGED](https://github.com/jupyter-naas/awesome-notebooks/assets/60933196/31baac32-be1c-4a68-95c0-9ba84b78e1f4)

**To Reproduce**
1. Go to [https://workspace.naas.ai/#](url)

**Current Developed Screen**
<img width="1680" alt="Screenshot 2023-06-13 at 3 44 54 PM" src="https://github.com/jupyter-naas/awesome-notebooks/assets/60933196/abfc033b-e95b-42a2-b59d-83e7dd4d6462">

**Comment by richagarg278** [comment]
@jravenel Check this issue.

**Comment by jravenel** [comment]
Agreed. This is minor issue but we shall keep that first design as reference
